app/models/page.rb

```diff
@@ -26,7 +26,7 @@ def only_page?
   private
 
   def generate_slug
-    self.slug = I18n.transliterate(title).parameterize
+    self.slug = I18n.transliterate(title.to_s).parameterize
```

Comment:
`.presence` в конце для порядка :)
```suggestion
    self.slug = I18n.transliterate(title.to_s).parameterize.presence
```